editor/components/block-list/index.js

```diff
@@ -261,5 +328,11 @@ export default connect(
 		onRemove( uids ) {
 			dispatch( { type: 'REMOVE_BLOCKS', uids } );
 		},
+		onReplace( uids, blocks ) {
+			dispatch( replaceBlocks( uids, blocks ) );
+		},
+		onChange( uid, attributes ) {
+			dispatch( updateBlockAttributes( uid, attributes ) );
+		},
```

Comment:
Tangent to the PR, but I really dislike `( dispatch ) => ( { onBlah( x, y, z ) { dispatch( doBlah( x, y, z ) ); } } )` when we have `{ onBlah: doBlah }`.